### ISSUE TITLE
Focus Trap When Exiting Fullscreen 

### ISSUE THREAD

**Issue by justinbansal** [sampotts/plyr#1665]
### Expected behaviour
When using the keyboard and exiting fullscreen, the expected behaviour is for you to be able to tab forward to the next element. This ends up being the video wrapper usually.
### Actual behaviour
Upon exiting fullscreen, you are stuck in a focus trap. I can't tab forward. If the video is playing, it is stuck here but if video stops, then it will tab forward but instead of going to the wrapper element next, it goes to the play button icon. From this point a tab forward goes to the play/pause button and through all the controls again. It just keeps cycling through this.

### Steps to reproduce
Tab through player (mine is within a Flickity slider), hit enter on fullscreen, hit escape to exit, then try to tab forward.

### Environment

- Browser: Chrome
- Version:
- Operating System: 
- Version:

**Comment by sampotts** [comment]
This should be resolved in v3.5.7. I've used your PR but also fixed the root issue and moved the focus trap logic over to the fullscreen class.
